test/Sentry.Unity.Editor.iOS.Tests/SentryXcodeProjectTests.cs

```diff
@@ -129,7 +129,7 @@ public void AddBuildPhaseSymbolUpload_CleanXcodeProject_BuildPhaseSymbolUploadAd
             xcodeProject.ReadFromProjectFile();
 
             var didContainUploadPhase = xcodeProject.MainTargetContainsSymbolUploadBuildPhase();
-            xcodeProject.AddBuildPhaseSymbolUpload(_fixture.Options.DiagnosticLogger);
+            xcodeProject.AddBuildPhaseSymbolUpload(_fixture.Options.DiagnosticLogger, new SentryCliOptions());
```

Comment:
Since `uploadSources` is `false` by default, should we add unit tests for the path that adds `--include-sources`?